src/services/session.ts

```diff
@@ -1,8 +1,9 @@
-import { Profile } from "../utilities/types";
```

Comment:
Still grabbing my old commits for session storage story. These commits are outdated and adding them back in would break things.

Reply:
**note: session storage broke in Josh's PR.  Bug fix story in pivotal.